src/java.base/windows/native/libjava/io_util_md.c

```diff
@@ -251,9 +268,13 @@ FD winFileHandleOpen(JNIEnv *env, jstring path, int flags)
     free(pathbuf);
 
     if (h == INVALID_HANDLE_VALUE) {
+        Trc_io_handleOpen_err(pathStr, access, sharing, disposition, flagsAndAttributes, GetLastError());
```

Comment:
This tracepoint and the one on line 276 should be conditional on `NULL != pathStr`.

Reply:
Similarly to `inet_ntop` I think it's better to print the tracepoint with the data available, even if the pathStr is printed as `<NULL>`.